tests/hpc/ww4_await_pxe_install.pm

```diff
@@ -0,0 +1,48 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Compute node of warewulf4 cluster
+#     Once controller is up and running, run a VM which is provisioned
+#     by the controller, and run checks.
+# Maintainer: Kernel QE <kernel-qa@suse.de>
+
+use Mojo::Base qw(hpcbase), -signatures;
+use testapi;
+use lockapi;
+use utils;
+use mm_tests;
+use POSIX 'strftime';
+
+sub run ($self) {
+    my $i = 0;
+
+    while ($i < 10) {
+        check_screen('pxe-start');
+        record_info "pxe0", "pxe0";
+        sleep(1);
+        send_key "ctrl-B";
+        sleep(20);
+        type_string("reboot", lf => 1);
+        save_screenshot();
+        $i++;
+    }
+    wait_still_screen 5;
+    while (1) {
+        unless (check_screen('ww4-ready')) {
+            record_info('Booting', '');
+            next;
+        }
+        last;
```

Comment:
What is the purpose of `wait_still_screen` and above booting loop? `assert_screen 'ww4-ready'` would work as well, as you just wait for https://aquarius.suse.cz/tests/16377#step/ww4_await_pxe_install/41

Reply:
removed